cmd/wallet-web/src/store/index.js

```diff
@@ -13,11 +13,12 @@ import credentials from './modules/credentials';
 import presentation from './modules/presentation';
 import user from './modules/user';
 import options from './modules/options';
+import store from './modules/store';
 import sharedMutations from 'vuex-shared-mutations';
 
 Vue.use(Vuex);
 
 export default new Vuex.Store({
-  modules: { connections, mode, mediator, credentials, presentation, user, options },
-  plugins: [sharedMutations({ predicate: ['setUserLoggedIn'] })],
+  modules: { connections, mode, mediator, credentials, presentation, user, options, store },
```

Comment:
can we have more specific name for this store, like locale ?

Reply:
I was thinking of it as a global state for the application, so that other things could be added to it later, when needed. But we could also just create a separate module for each of the things that we need to store in a global state. What do you think @talwinder50 @rolsonquadras @sudeshrshetty?

Reply:
Addressed